packages/typespec-azure-resource-manager/lib/common-types/managed-identity.tsp

```diff
@@ -33,10 +33,23 @@ model ManagedServiceIdentity {
   type: ManagedServiceIdentityType;
 
   @doc("The identities assigned to this resource by the user.")
-  userAssignedIdentities?: UserAssignedIdentities;
+  userAssignedIdentities?: Record<UserAssignedIdentity>;
 }
 
 /** The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.", */
+@armCommonDefinition(
```

Comment:
What was that doing before

Reply:
If people use the ManagedServiceIdentityProperty mix-in, then the common-types are referenced before you get here.  This is only to pave the way for brownfield APIs that reference this type directly (there are around 10, I believe)